docs/docs/how-tos/nebari-destroy.md

```diff
@@ -0,0 +1,180 @@
+
+:::warning
+If you have any data stored on your Nebari cluster you would like to keep - either files on the filesystem, conda environments in conda-store or an exported `json` of the users and groups from Keycloak - now is the time to back those up **off of the cluster**. 
+:::
+
+```bash
+nebari destroy -h
+```
+
+```bash
+ Usage: nebari destroy [OPTIONS]
+
+ Destroy the Nebari cluster from your nebari-config.yaml file.
+
+╭─ Options ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *  --config          -c      TEXT  qhub configuration file path [default: None] [required]                                │
+│    --output          -o      TEXT  output directory [default: ./]                                                         │
+│    --disable-render                Disable auto-rendering before destroy                                                  │
+│    --disable-prompt                Destroy entire Nebari cluster without confirmation request. Suggested for CI use.      │
+│    --help            -h            Show this message and exit.                                                            │
+╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+:::note
+The above command will not delete your `nebar-config.yaml` or the rendered files thus a re-deployment with `nebari deploy` is possible afterwards. 
+:::
+
+Once you've decided to destroy your cluster and have read through the `nebari destroy --help` command, you are you now ready.
+
+```bash
+nebari destroy -c nebari-config.yaml
+```
+
+The terminal will prompt you to press `enter` to verify that you wish to destroy your Nebari cluster.
+
+:::info
+This destruction process can take up to 30 minutes for some cloud providers.
+:::
+
+If the destruction is successful, you will see the following output:
+
+```bash
+[terraform]: Nebari properly destroyed all resources without error
+```
+
+Congratulations! You have successfully destroyed your Nebari deployment and the associated resources!
+
+## Manually destroying the resources
+
+### Amazon Web Services (AWS)
+
+Unfortunately of all the cloud providers supported, AWS seems to make it the most difficult to destroy lingering resources. They often times requiring resources be destroyed in a particular order as well.
+
+#### Destroy resources from the AWS console
+
+Here, we outline how to find and destroy resources from the AWS console. 
+
+1. Sign into your AWS account and in the search bar type `Tag Editor`
+
+<img src="/img/how-tos/aws_tag_editor_1.png" alt="Search for Tag Editor in the AWS console" width="700"/>
+
+1. From the `Resource Groups & Tag Editor` page, navigate to `Tag Editor` from the left most panel
+
+<!-- ![Navigate to the Tag Editor in the AWS console](../../static/img/how-tos/aws_tag_editor_2.png) -->
```

Comment:
I think we remove this comment because the following line adds this image?

Reply:
Removed :)